changelog/7338.bugfix.md

```diff
@@ -0,0 +1 @@
+Sender id is correctly set when copying the tracker and sending it to the action server (instead of sending the `default` value)
```

Comment:
what do you think?
```suggestion
Sender ID is correctly set when copying the tracker and sending it to the action server (instead of sending the `default` value). This fixes a problem where the action server would only retrieve trackers with a `sender_id` `default`.
```